test/test_pg_client_store.js

```diff
@@ -0,0 +1,149 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import db  from '../src/pg_store';
+import should from 'should';
+
+describe('db for api clients', () => {
+  it('1. should succeed to clear db', done => {
+    db.clear().then(done);
+  });
+
+  it('2. should be an empty db', done => {
```

Comment:
nit: Do not add numbers to the test description, please. Changing one, probably means that we need to change the others ;)